gnulib: symlink include/ to lib/

teto opened the pull request:
so that some programs can find the headers automatically instead of
adding -I${gnulib}/lib



- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.37865&attempt_id=03ca7ed9-8c61-48f5-88d0-181d4540e084)

Attempted: gnulib

<details><summary>Partial log (click to expand)</summary><p>

```
updateAutotoolsGnuConfigScriptsPhase
Updating Autotools / GNU config script to a newer upstream version: ./build-aux/config.sub
Updating Autotools / GNU config script to a newer upstream version: ./build-aux/config.guess
configuring
no configure script, doing nothing
building
build flags: SHELL=/nix/store/3gg2p6n4kv2f0lsxd41f5iz1ivkbzyzr-bash-4.4-p19/bin/bash
make: Nothing to be done for 'all'.
installing
/nix/store/6rfmkma0nw1wxvy7g7f8g8mqqr9ihj4p-gnulib-20180226
```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.37865&attempt_id=00fd4fef-5a61-4b84-92db-39a25f7345e4)

Attempted: gnulib

<details><summary>Partial log (click to expand)</summary><p>

```
unpacking source archive /nix/store/8xzcy8ldivf3l7dms88h5gzfadlnrgw9-gnulib-0bec5d5
source root is gnulib-0bec5d5
patching sources
configuring
no configure script, doing nothing
building
build flags: SHELL=/nix/store/q1g0rl8zfmz7r371fp5p42p4acmv297d-bash-4.4-p19/bin/bash
make: Nothing to be done for 'all'.
installing
/nix/store/232hm55bb26jyh7ll6f64zianwsfirfi-gnulib-20180226
```
</p></details>

teto committed:
gnulib: symlink include/ to lib/

so that some programs can find the headers automatically instead of
adding -I${gnulib}/lib
